src/watchmaker/__init__.py

```diff
@@ -152,6 +152,14 @@ class Arguments(dict):
 
                 ou_path="OU=Super Cool App,DC=example,DC=com"
 
+        pip_install: (:obj:`str`)
```

Comment:
I don't think we need to update the cli with this arg at the moment. Cli args come in as strings, and we are expecting this to be a list, so this won't work on the cli anyway. And that's fine for now, as this is most likely to be managed solely through the config file.